playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -99,6 +127,8 @@
       env: "{{ env }}"
       host-type: "{{ host_type }}"
       env-host-type: "{{ env_host_type }}"
+      node-type: "{{ node_type }}"
```

Comment:
Could we call this "sub-type" instead, that way we can use it with other host-types in the future if we need to?